.devcontainer/Dockerfile

```diff
@@ -7,12 +7,12 @@ RUN if [ -f "/tmp/conda-tmp/environment.yml" ]; then umask 0002 && /opt/conda/bi
     && rm -rf /tmp/conda-tmp
 
 # Install additional OS packages.
-RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends curl gnupg lsb-release bash-completion \
-    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+#     && apt-get -y install --no-install-recommends curl gnupg lsb-release bash-completion \
```

Comment:
that's a shame
bash-completion is the one I think most will miss (especially for make file completion)

I'll live

Reply:
Yea so there are options here - main one being that if we find things we can't/won't download in the TRE that we need, then we can pre-create a devcontainer image, and stash that in an 'internal' ACR, and then reference that in the Dockerfile.

For now i've commented stuff out as I don't _think_ any of it is required for the use case, but might be wrong. Gets us moving at least

Reply:
Cool